docs/user-guide/manage-data/overview.md

```diff
@@ -312,6 +312,16 @@ If a table has its own TTL policy,
 it will take precedence over the database TTL policy.
 Otherwise, the database TTL policy will be applied to the table.
 
+The value of 'ttl' can be one of duration(like `1hour 12min 5s`), `immediate` or `forever`. See details in [CREATE](/reference/sql/create.md#create-a-table-with-ttl) statement.
+
+If you want to remove the ttl policy, you can use the following SQL
+
+```sql
+ALTER TABLE monitor UNSET 'ttl';
+-- or for database
+ALTER DATABASE db UNSET 'ttl';
```

Comment:
```suggestion
```

there is no database example in this guide

Reply:
> there is no database example in this guide

didn't get it, there is a `CREATE DATABASE test WITH ('ttl'='7d');` few lines up